api/app.ts

```diff
@@ -30,6 +30,9 @@ app.post('/changePassword', changePasswordEndpoint);
 import accountCreate from './accounts/create';
 app.post('/accounts/create', accountCreate);
 
+import resetUsername from './accounts/account-update-username';
+app.post('/accounts/resetUsername', resetUsername);
```

Comment:
endpoint should hit the endpoint `accounts/changeUserName`.
```suggestion
app.post('/accounts/changeUsername', changeUsername);
```

Reply:
fixed.